Tasks/VSMobileCenterTest/vsmobilecentertest.ts

```diff
@@ -86,123 +195,33 @@ async function run() {
         tl.checkPath(cliPath, "mobile-center");
 
         let debug: boolean = tl.getBoolInput('debug', false);
-
+        let prepareTests: boolean = tl.getBoolInput('enablePrepare', true);
```

Comment:
This is debatable so I will leave it up to you.  Personally I think checkboxes by definition are optional, so I always use "false" in getBoolInput() -- it's really hard to "require" a user to must take a look at the checkbox, and determine if the value should be checked or not.  I don't think this will make a difference either way as build agent will always populate the environment variable with "true" or "false".  This will only make a difference in writing tests -- if it's required we must set the input, otherwise we can leave it out.